src/DssDeploy.sol

```diff
@@ -259,7 +261,7 @@ contract DssDeploy is DSAuth {
         vow.rely(address(cat));
     }
 
-    function deployEnd() public auth {
+    function deployEnd(address gov, address sun, uint256 min) public auth {
```

Comment:
I'd like to rename it to `deployShutdown` as it also includes the `esm` module now and it would be more descriptive as the other functions (excepting maybe the `pause` one).